Makefile

```diff
@@ -67,7 +67,7 @@ integration-cleanup:
 	./integration/test-cleanup.sh
 .PHONY: integration-cleanup
 
-integration: build $(GOBIN)/commander 
```

Comment:
My code editor automatically removes trailing white spaces. I can revert these changes if need be.

Reply:
I think this is fine as it doesn't add too much noise. Thanks! Scout rule:)